apps/tasks/management/commands/generate_tasks.py

```diff
@@ -0,0 +1,39 @@
+import random
+
+from faker import Faker
+
+from apps.accounts.models import User
+
+from django.core.management.base import BaseCommand
+
+from apps.tasks.models import Task
+
+
+class Command(BaseCommand):
+    help = 'Create random tasks for all users in database.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('instances', type=int, help='Number of task instances to be created.')
+
+    def handle(self, *args, **kwargs):
+        fake = Faker()
+        total = kwargs['instances']
+        users_id = list(User.objects.values_list('id', flat=True))
+        if users_id:
+            instances = []
+            for i in range(total):
```

Comment:
Sfat: cind nu folosesti variablila din for da-i numele underscore ( for _ in range )